Lib/turtle.py

```diff
@@ -3382,6 +3404,26 @@ def filling(self):
         """
         return isinstance(self._fillpath, list)
 
+    @contextmanager
+    def fill(self):
+        """Create a filled shape.
+
+        No argument.
+
+        This function sets up a context manager that will automatically end
+        filling once exited.
```

Comment:
1. I don't think it is needed to inform the user that this function does not take arguments.
2. I'd prefer if the docstring title mentions that this is a context manager.

How about something like this:

```suggestion
        """A context manager for filling a shape.

        Implicitly ensures the code block is wrapped with
        begin_fill() and end_fill().
```